less/patternfly-react.less

```diff
@@ -1,3 +1,62 @@
 /**
   Patternfly React Specific Extensions
 */
+@import '~patternfly/dist/less/variables';
+
+/** Following is used by SerialConsole */
```

Comment:
What about introducing `.less` files local to the component? Means like `src/SerialConsole/styles.less` .
They would be listed in `less/patternfly-react.less` for import.

Reply:
I'd be ok with that if that's what we all agree on. Still need the sass version too though.

Reply:
Could you use a separate file here as well? Add this to `serialConsole.less` and import the file here.